src/python/pants/backend/go/target_type_rules.py

```diff
@@ -173,9 +173,9 @@ class InjectGoExternalPackageDependenciesRequest(InjectDependenciesRequest):
     inject_for = GoExtModPackageDependencies
 
 
-# TODO(12761): This duplicates first-party dependency inference but that other rule cannot operate on_go_ext_mod_package
-# targets since there is no sources field in a _go_ext_mod_package. Consider how to merge the inference/injection
-# rules into one. Maybe use a private Sources field?
+# TODO(#12761): This duplicates first-party dependency inference but that other rule cannot operate
```

Comment:
The `on` in `operate on` went away since there was no space but don't block PR on this comment.